SoapySDRPlay.hpp

```diff
@@ -295,9 +296,9 @@ class SoapySDRPlay: public SoapySDR::Device
     static std::unordered_map<std::string, sdrplay_api_DeviceT*> selectedRSPDevices;
 
     // RX callback reporting changes to gain reduction, frequency, sample rate
-    int gr_changed;
-    int rf_changed;
-    int fs_changed;
+    volatile int gr_changed;
```

Comment:
Why volatile? A simple `std::atomic<int>` should be sufficient. When accessing, `std::memory_order_relaxed` should also be fine.

Reply:
Mostly because "volatile" feels simpler to me than all the funny alphabet soup introduced in modern C++